flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducer.java

```diff
@@ -1355,6 +1353,7 @@ String getTransactionalId() {
         FlinkKafkaInternalProducer<byte[], byte[]> producer =
                 initTransactionalProducer(transactionalId, true);
         producer.initTransactions();
+        System.out.println("NEW PRODUCER " + transactionalId);
```

Comment:
Was this intentionally comitted? All the other code uses `LOG`.

Reply:
🤦 leftover from debugging, thanks